storage_drivers/solidfire_san.go

```diff
@@ -202,7 +202,9 @@ func (d *SolidfireSANStorageDriver) Create(name string, opts map[string]string)
 
 	log.Debugf("GetVolumeByName: %s, %d", name, d.TenantID)
 	log.Debugf("Options passed in to create: %+v", opts)
-	v, err := d.Client.GetVolumeByName(name, d.TenantID)
+        // SolidFire can only handle letters, numbers and - in volume name
+        theName := strings.Replace(name, "_", "-",-1)
```

Comment:
Since we are changing the name from what was requested you might want to consider adding a log statement indicating the old and the new name to help with debug-ability.